template.yaml

```diff
@@ -20,6 +20,12 @@ Parameters:
     Type: String
     Description: The ARN of the subscription endpoint to send logs to splunk
     Default: "arn:aws:logs:eu-west-2:885513274347:destination:csls_cw_logs_destination_prodpython-2"
+  SlackWebhookUrl:
```

Comment:
We can't pass parameters like this because of the way secure pipelines is structured. Probably best to fetch at runtime by leaving unset.